lua/autorun/server/sv_hideholos.lua

```diff
@@ -0,0 +1,43 @@
+-- Updates the visibility of all holos for a specific player
+local function updateHoloVisibility( ply )
+    for _, holo in pairs( ents.FindByClass( "gmod_wire_hologram" ) ) do
+        holo:SetPreventTransmit( ply, ply.cfc_holosDisabled )
+    end
+end
+
+-- Initialize the player's cfc_holosDisabled variable and hide all existing holos
+local function initializePlayer( ply )
+    ply.cfc_holosDisabled = true
+
+    updateHoloVisibility( ply )
+end
+
+hook.Add( "PlayerInitialSpawn", "CFC_HideHolos_initalizePlayer", initializePlayer )
+
+-- Update a player's holo visibility using chat commands
+local function getCommands( ply, text )
+    if text == "!displayholos" then
+        ply.cfc_holosDisabled = false
+
+        updateHoloVisibility( ply )
+    elseif text == "!hideholos" then
+        ply.cfc_holosDisabled = true
+
+        updateHoloVisibility( ply )
+    end
```

Comment:
This is silly and unnecessary, but you could try something like this:

```suggestion
    local states = { ["!displayholos"] = false, ["!hideholos"] = true }
    if states[text] == nil then return end

    ply.cfc_holosDisabled = states[text]
    updateHoloVisibility( ply )
```